tests/test_qnode.py

```diff
@@ -458,22 +473,24 @@ def func(x, y):
         y = pnp.array(0.54, requires_grad=True)
 
         res = qn(x, y)
+        tape = qml.workflow.construct_tape(qn)(x, y)
```

Comment:
I'd actually leave this test case as is, but put `with pytest.warns(qml.PennyLaneDeprecationWarning)` around accessing the tapes.  Then we can delete this test when we remove `QNode.tape`. 

This test seems to be explicitly about the `QNode.tape` proprety, so want to make sure we are deleting it when we delete `QNode.tape`.